src/pages/using-spark/guides/alt-text.mdx

```diff
@@ -0,0 +1,337 @@
+  Alternative Text
+</h1>
+
+UX Writers, designers and developers can use this **Alternative Text Guide** as an instruction to writing style, usage, text for copy, documentation, reference information, and copy inside Rocket Products.
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvl" />
+
+<h3 class="sprk-b-TypeDisplayThree sprk-u-mbm sprk-u-Measure">Main Takeaways</h3>
+
+- Adding alternative text ensures user with disabilities can understand non-text content.
+- Describe the image without altering the meaning of the screen or referencing the word “image.”
+- Consider how you would describe that image to someone without them having to visually see the image.
+- Decorative images don't need alternative text.
+- Use a null attribute (or empty string) for non-semantic or decorative images, i.e., those that are for ambiance or other non-semantic purposes.
+
+#### Accessibility Benefits
+
+- Screen readers read alt-text in place of images. This allows the image to be accessible to users with visual impairment.
+- When an image fails to load or if content is blocked, the alt-text will appear in it's place.
+- It gives images a text to display so screen readers can read it without altering the meaning of the screen.
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvs" />
+
+### Table Of Contents
+
+<ul>
+  <li>
+    <a href="#what-is-alternative-text">What Is Alternative Text</a>
+  </li>
+  <li>
+    <a href="#images-and-graphics">Images and Graphics</a>
+  </li>
+  <ul class="sprk-b-List sprk-b-List--indented sprk-u-Measure sprk-u-mvs">
+    <li>
+      <a href="#images">Images</a>
+    </li>
+    <li>
+      <a href="#icons">Icons</a>
+    </li>
+    <li>
+      <a href="#svgs">SVGs</a>
+    </li>
+  </ul>
+  <li>
+    <a href="#writing-effective-alt-text-copy">
+      Writing Effective Alt Text Copy
+    </a>
+  </li>
+  <ul class="sprk-b-List sprk-b-List--indented sprk-u-Measure sprk-u-mvs">
+    <li>
+      <a href="#guidelines">Guidelines</a>
+    </li>
+    <li>
+      <a href="#examples">Examples</a>
+    </li>
+  </ul>
+  <li>
+    <a href="#accessibility">Accessibility</a>
+  </li>
+    <ul class="sprk-b-List sprk-b-List--indented sprk-u-Measure sprk-u-mvs">
+    <li>
+      <a href="#resources">Resources</a>
+    </li>
+  </ul>
+</ul>
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvn" />
+
+## What Is Alternative Text
+
+Alternative text or commonly known as **alt-text** is a word or phrase that describes an image’s appearance and function.
+
+> The main purpose of the <a href="https://www.w3.org/TR/WCAG21/#non-text-content">alt tag</a> is to support accessibility, so screen readers and other assistive technology users are able to make full use of the content.
+
+<h3 class="sprk-b-TypeDisplayFour sprk-u-mbm sprk-u-Measure">Syntax</h3>
+
+The text inside the alt tag provide the user with a description of the image.
+
+```
+<img src="image_of_puppies.png" alt="Four mini black color goldendoodles">
```

Comment:
Alt text reads awkwardly, try:
"Four black Goldendoodle puppies"

Reply:
@RVMendoza - thank you, feedback has been implemented